app/assets/locales/locale-en.json

```diff
@@ -1708,8 +1708,48 @@
+                "text_legend": "Collateral Ratio (CR): The CR defines how much percent of the generated SmartCoin value is covered by collateral.\nMaintenance Collateral ratio (MCR): To guarantee that the SmartCoin has enough backing collateral and to account for volatility, the MCR is enforced when creating the position."
+            },
+            "steps_risks": {
+                "title": "Risks",
+                "title_within": "What are the risks involved in borrowing?",
+                "text": "As long as you monitor your position and make sure that the collateral ratio (CR) stays above the maintenance collateral ratio (MCR), your position will not be liquidated. If the value of the collateral drops and your position gets close to MCR, you can lock up more collateral or pay back some of the debt to increase the CR. Failure to remain above MCR results in force liquidation of your position (margin call). This means that your collateral is being sold automatically on the respective market to reduce your debt, with a penalty to ensure the order is filled (see MSSR). Any leftover collateral is returned to you after the full debt is paid. You can choose to only allow partial selling of your position by setting the target collateral ratio (TCR). If your position can not be force liquidated due to extreme market conditions and the CR drops below the maximum short squeeze ratio (MSSR), the SmartCoin automatically enters global settlement state and you lose your position instantly.",
+                "text_legend": "Target collateral ratio (TCR): If your position is liquidated the system automatically puts an order on the market buying back the full debt. If TCR is set, the order will only sell as much debt as needed such that the CR of your position is the TCR. \n Maximum short squeeze ratio (MSSR): When your position is force liquidated, it is being sold below the current feed price to ensure the order is filled. The MSSR defines how much below feed price. \n Global Settlement: A globally settled SmartCoin can no longer be generated by opening a Collaterized Debt Position while instead holders of it can settle it instantly for a fixed Global Settlement Price."
+            },
+            "steps_costs": {
+                "title": "Settlement",
+                "title_within": "",
+                "text": "Some longer text Some longer text Some longer text Some longer text Some longer text Some longer text Some longer text Some longer text Some longer text "
+            }
         },
         "direct_debit": {
             "description": "Setup a direct debit mandate for recurring payments. The mandate allows another account to transfer funds out of your account, with a limit on the amount for defined time intervals.",
```

Comment:
```suggestion
            "description": "Setup a direct debit mandate for recurring payments (all parties are required to have a Bitshares account). The mandate allows another account to transfer funds out of your account, with a limit on the amount for defined time intervals.",
```

Reply:
Yes Sir, noted. I only mention it because in 2014 I was very excited to see Bitshares was capable of recurring payments. However, it was not immediately clear to me that both parties needed to have a Bitshares account. I.E. recurring employment, bill pay, etc can not be direct to a merchant unless they also have a Bitshares account. In 2014 merchants would have been leary in comparison to advances in the Blockchain industry today.

Reply:
Also, consider adding memo (i.e. receipt, description of mandate, what is this for? how much? when?, merchant correspondence, etc)
Additionally, consider adding a memo that acts as a personal note to self (i.e. recurring payment to X, bill pay to X, etc)